install_config/install/docker_registry.adoc

```diff
@@ -189,9 +200,8 @@ Once the quota is exceeded, an image push will fail with:
 Failed to push image: unknown: requested access to the resource is denied
 ----
 
-You may then
-link:../../admin_guide/pruning_resources.html#pruning-images[prune obsoleted images]
-to retrieve some space back.
+Then, you may link:../../admin_guide/pruning_resources.html#pruning-images[prune
+obsoleted images] to retrieve some space back.
```

Comment:
Unless I'm seeing this wrong in this GitHub UI, I would not put a line break between "prune" and "obsoleted images". This comes back to the discussion we had in this week's meeting about hard wrapping, where you don't want to hard wrap in the middle of a term, or link.